src/scrapers/tools/geocoder.py

```diff
@@ -10,29 +10,32 @@
 from models.initiatives import InitiativeImport
 
 
+def get_logger() -> logging.Logger:
+    return logging.getLogger(__name__)
+
+
 class Geocoder:
 
     def __init__(self):
         self.geo_locator = Nominatim(user_agent="code-for-nl-covid-19")
 
-    def get_logger(self) -> logging.Logger:
-        return logging.getLogger(__name__)
-
     def batch(self, feature_type=None):
-        logger = self.get_logger()
+        logger = get_logger()
         db = Db()
         # Default concat function to sqlite
-        concat_func = func.group_concat(InitiativeImport.id.distinct()).label('id_array')
+        concat_func = func.group_concat(InitiativeImport.id.distinct()).label("id_array")
 
-        if db.session.bind.driver != 'pysqlite':
+        if db.session.bind.driver != "pysqlite":
             # Assume postgres
-            concat_func = func.array_agg(InitiativeImport.id, type_=ARRAY(Integer)).label('id_string')
+            concat_func = func.array_agg(InitiativeImport.id, type_=ARRAY(Integer)).label("id_string")
 
         location_set = db.session.query(
             InitiativeImport.location,
             concat_func
         ) \
             .filter(InitiativeImport.location.isnot(None)) \
+            .filter(InitiativeImport.longitude.isnot(None)) \
```

Comment:
Dit is te vroeg. Want nou krijgen deze items ook geen osm_address meer. De oude code had een check bij het setten van lat/long na het ophalen van de geocode.

Zodoende stelde ik voor om twee update queries te schrijven in de geocode functie. één die enkel osm_address update in de set en één die ook lat/long update als bv lat leeg is.